tasks/hyrax-metadata-updates/index.js

```diff
@@ -0,0 +1,290 @@
+'use strict';
+
+const cumulusMessageAdapter = require('@cumulus/cumulus-message-adapter-js');
+const { InvalidArgument } = require('@cumulus/errors');
+
+const get = require('lodash.get');
+const _ = require('lodash/core');
```

Comment:
eslint complains on this one. says `Import Lodash methods only from method packages`

Reply:
Done.